### DISCUSSION TITLE
VIAへの対応について

### DISCUSSION THREAD

**Discussion by Hideck-Github** [Yowkees/keyball#339] [Q&A]
お世話になっております。
Keyball61をVIAで利用しようと、via.jsonをデザインモードでアップロードしたことろ、
以下のようなエラーになりました。
Remapが無くなる可能性もあるので、どちらの対応をお考えなのか教えていただけますでしょうか。

![image](https://github.com/Yowkees/keyball/assets/70126181/358e6dd2-6a35-4317-877c-6960c878f2e5)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
現在、最新0.22のQMKに対応したKeyball用ファームウェアを作成しておりますので、最新バージョンリリース後に見直したいと思いますので、お願いいたします。

**Comment by sjackwu** [comment]
> お世話になっております。 Keyball61をVIAで利用しようと、via.jsonをデザインモードでアップロードしたことろ、 以下のようなエラーになりました。 Remapが無くなる可能性もあるので、どちらの対応をお考えなのか教えていただけますでしょうか。
> 
> ![image](https://private-user-images.githubusercontent.com/70126181/267163386-358e6dd2-6a35-4317-877c-6960c878f2e5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3OTAxMDMsIm5iZiI6MTcyODc4OTgwMywicGF0aCI6Ii83MDEyNjE4MS8yNjcxNjMzODYtMzU4ZTZkZDItNmEzNS00MzE3LTg3N2MtNjk2MGM4NzhmMmU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEzVDAzMjMyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3ZjIyMzk0NTM1YjIwZDVmZDliMTNlZjMyYzU5YjEyZTA0YTYyZjIwNDM4NmYxOTQzY2Q0NzhjZmQ4ZTk1YmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ZbWQUQ08cfngFXPbF2T_cEbC0olAkvLbucX9IpVd1OM)

I did the same thing on VIA for a different reason. REMAP has limited support for hold/tap function, for example, I cannot set "tap: Q; hold: cmd-Q" or "tap: m; hold: +".

I learned from issue #556 that VIAL support is not possible, so I guess that VIA support is my last hope? 



<p align="right">A novice keyball44 user in Taiwan</p>
